.gitignore

```diff
@@ -16,6 +16,9 @@
 .AppleDouble
 .LSOverride
 
+# IDE ignores
+.idea/
+
```

Comment:
we recommend you place this in your own personal gitignore since this is user specific

Reply:
Good point, I just figured out I could use `.git/info/exclude` locally...